daemons/ipa-kdb/ipa_kdb.c

```diff
@@ -628,6 +675,14 @@ static krb5_error_code ipadb_init_module(krb5_context kcontext,
         goto fail;
     }
 
+    /* Enforce PAC ticket signature verification if supported by all KDCs */
+    kerr = should_support_pac_tkt_sign(kcontext, &pac_tkt_sign_supported);
```

Comment:
Because this check happens at KDC startup time only, all KDC services should be restarted after all servers are updated to support PAC ticket signature, in order to execute this test again once the LDAP `ipaConfigString` attribute is updated by all KDCs.